app/views/users/show.html.erb

```diff
@@ -8,9 +9,16 @@
 </div>
 
 <div class="user-posts">
+<% @recent_posts.each do |post| %>
+  <%= render post %>
+    <% end %>
+  
```

Comment:
- [ ] The same thing here, Kindly also preview the five recent comments of the post here (post.most_recent_five_comments)